stl/inc/utility

```diff
@@ -44,7 +44,7 @@ void iter_swap(_FwdIt1 _Left, _FwdIt2 _Right) { // swap *_Left and *_Right
 }
 
 // FUNCTION TEMPLATE swap
-template <class _Ty, size_t _Size, class>
+template <class _Ty, size_t _Size, int>

```

Comment:
I think you need to repeat the constraint, otherwise there's a mismatch between declaration and definition.

```suggestion
template <class _Ty, size_t _Size, enable_if_t<_Is_swappable<_Ty>::value, int>>
```

Reply:
This seems to not be needed, only the other.

Reply:
I changed it to correctly use the constraint